src/components/AppComponents/Wall/Wall.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from "@testing-library/react";
+
+import { Wall } from "./";
+
+describe(Wall, () => {
+  it("hides the given content", () => {
+    render(
+      <Wall>
+        <div data-testid="canary" />
+      </Wall>,
+    );
+
+    const content = screen.getByTestId("content");
+
+    expect(content).toHaveStyle(`opacity: 0`);
+    expect(content).toHaveAttribute("aria-hidden", "true");
```

Comment:
small thing but a `not.toBeVisible` check would be nice here as it checks a bunch of different ways of 'hiding' elements for us, seems unlikely we'd change the implementation of this but if we did it would still pass the test!